pkg/cmd/util/downloadrequest/downloadrequest.go

```diff
@@ -47,18 +47,18 @@ func Stream(client arkclientv1.DownloadRequestsGetter, name string, kind v1.Down
 		},
 	}
 
-	req, err := client.DownloadRequests(v1.DefaultNamespace).Create(req)
+	req, err := client.DownloadRequests(namespace).Create(req)
 	if err != nil {
 		return errors.WithStack(err)
 	}
-	defer client.DownloadRequests(v1.DefaultNamespace).Delete(req.Name, nil)
+	defer client.DownloadRequests(namespace).Delete(req.Name, nil)
 
 	listOptions := metav1.ListOptions{
 		//TODO: once kube-apiserver http://issue.k8s.io/51046 is fixed, uncomment
```

Comment:
feel like fixing this while you're in here?

Reply:
I can update the comment, but we can't use a field selector until Kubernetes v1.9.0 is the minimum supported version.

Reply:
good point :)